framework/dependencies.yml

```diff
@@ -42,7 +42,8 @@ require:    &allDependencies
     - oauth.signpost -> signpost-core 1.2.1.2
     - org.apache.geronimo.specs -> geronimo-servlet_2.5_spec 1.2
     - org.apache.ivy -> ivy 2.4.0
-    - org.bouncycastle -> bcprov-jdk15 1.46
+    - org.bouncycastle -> bcprov-jdk15on 157
```

Comment:
True. But on the original homepage you will find `bcprov-jdk15on-157.jar` without a period dot between numbers. see https://www.bouncycastle.org/latest_releases.html
I am not sure if all Play! dependencies are really from maven. That would be nice because of checksums and sometimes additional signatures.